bin/cordova

```diff
@@ -29,7 +29,11 @@ cli(process.argv).catch(err => {
         const errorOutput = typeof err === 'string' ? err : util.inspect(err);
         throw new CordovaError('Promise rejected with non-error: ' + errorOutput);
     }
-    process.exitCode = err.code || 1;
+    if (typeof err.code !== "number") {
```

Comment:
I think it's a good change to make a fallback. It just needs to use our code style, which is to use single quotes for strings.

```suggestion
    if (typeof err.code !== 'number') {
```

should satisfy our lint. After applying the change you can do `npm test` to confirm, or you can commit/push to let the PR checks to run.